cmd/kubeadm/app/util/etcd/testing/mock_etcd_cluster.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Generated code, generated via: `mockgen -destination=./testing/mock_etcd_cluster.go -package=testing go.etcd.io/etcd/clientv3 Cluster`
+// Edited by hand for boilerplate and gofmt.
+// TODO, this should be autogenerated/autoupdated by scripts.
+
+// Package testing is a generated GoMock package.
+package testing
+
+import (
+	context "context"
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+// MockCluster is a mock of Cluster interface.
+type MockCluster struct {
+	ctrl     *gomock.Controller
+	recorder *MockClusterMockRecorder
+}
+
+// MockClusterMockRecorder is the mock recorder for MockCluster.
+type MockClusterMockRecorder struct {
+	mock *MockCluster
+}
+
+// NewMockCluster creates a new mock instance.
+func NewMockCluster(ctrl *gomock.Controller) *MockCluster {
+	mock := &MockCluster{ctrl: ctrl}
+	mock.recorder = &MockClusterMockRecorder{mock}
+	return mock
+}
+
+// EXPECT returns an object that allows the caller to indicate expected use.
+func (m *MockCluster) EXPECT() *MockClusterMockRecorder {
```

Comment:
surprising that the k/k linters are happy with this function's uppercase name :thinking: